find-mmc-bypartlabel

```diff
@@ -34,18 +34,21 @@ while [ ! -d /sys/class/block/mmcblk0p* ] && [ ! -d /sys/class/block/sda ] ; do
     echo "find-mmc-bypartlabel: Waiting for /sys/class/block/mmcblk0p*..." > /dev/kmsg
```

Comment:
ie, here above have while [ ! -f /sys/class/block/mmcblk0p*/uevent ] && ... ?

Reply:
That checks for the device to exist. If we check for the partitions here it would break as this would expand in all found partitions at the time.

Reply:
This should be ok